nshimiyejayd/my-brand
# #5D0IuRMz Implementing Home page

#### What does this PR do?
This PR implements Home page

#### Description of Task to be completed?
- Implement home page

#### How should this be manually tested?
- Clone the  [repo](https://github.com/nshimiyejayd/my-brand)
- Checkout branch `feat-implement-home-page-5D0IuRMz`
- Open index.html in your browser to check Home page

#### What are the relevant pivotal tracker stories?
[#5D0IuRMz](https://trello.com/c/5D0IuRMz/18-home-page)
